test/binary-test.js

```diff
@@ -150,11 +156,26 @@ buster.testCase('Binary API', {
               assert.equals(value.toString(), 'a' + key + 'value')
               callback()
             } else {
-              common.checkBinaryTestData(value, callback)
+              checkBinaryTestData(value)
+              callback()
             }
           })
         }, done)
       })
     }.bind(this))
   }
 })
+
+function loadBinaryTestData () {
+  return Buffer.from('0080c0ff', 'hex')
+}
+
+function checkBinaryTestData (testData) {
+  assert.equals(md5sum(testData), binaryTestDataMD5Sum)
```

Comment:
We could get rid of `binaryTestDataMD5Sum` as well.

Reply:
These functions could now also be renamed to `loadData()` and `checkData()`, since binary is implied within this file.

Reply:
> You mean by using buf.equals instead?

I meant rather doing:

```js
function checkData (buf) {
  assert.equals(md5sum(buf), md5sum(loadData()))
}
```

Reply:
It would mean we don't cache the md5 sum of the test data, so a bit slower, but cleaner.

Reply:
Why do we need to compute a checksum if we can just compare the buffers?

Reply:
> Why do we need to compute a checksum if we can just compare the buffers?

That's even better. Didn't know about `buf.equals()`!